patches/third_party-blink-renderer-core-dom-element.cc.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/third_party/blink/renderer/core/dom/element.cc b/third_party/blink/renderer/core/dom/element.cc
+index 8a7abaeb53db0fdbe3f1ba9d52c7a2770ad9438b..1ad787c8cc0110ce99089c9bd8eddcab0cebfce5 100644
+--- a/third_party/blink/renderer/core/dom/element.cc
++++ b/third_party/blink/renderer/core/dom/element.cc
+@@ -162,6 +162,10 @@
+ #include "third_party/blink/renderer/platform/wtf/text/string_builder.h"
+ #include "third_party/blink/renderer/platform/wtf/text/text_position.h"
+ 
++#if defined(BRAVE_CHROMIUM_BUILD)
```

Comment:
this kind of stuff should be added through chromium_src override, but I think there is probably a much easier way to do this that doesn't require patching. Investigating now